admin/build.sbt

```diff
@@ -0,0 +1,46 @@
+lazy val piezoAdmin = (project in file(".")).enablePlugins(PlayScala)
+
+bashScriptExtraDefines ++= Seq(
+  "mkdir -m 664 /var/run/piezo-admin/",
+  s"chown ${(daemonUser in Linux).value} /var/run/piezo/admin",
+  s"chown ${(daemonGroup in Linux).value} /var/run/piezo/admin",
+  """JAVA_OPTS="-Dpidfile.path=/var/run/piezo-admin/piezo-admin.pid $JAVA_OPTS"""",
```

Comment:
By default, it will use org/quartz/quartz.properties.

I changed the file to that location (from its original quartz.properties) rather than set a default system property. I think that's better.